noise2same/decode_head.py

```diff
@@ -0,0 +1,162 @@
+from abc import ABCMeta, abstractmethod
+
+import torch
+import torch.nn as nn
+from .ops import resize
+
+
+class ConvBlock(nn.Sequential):
+
+    def __init__(self,
+                 in_channels: int,
+                 out_channels: int,
+                 kernel_size: int,
+                 padding: int = 0,
+                 conv_bias: bool = False):
+        conv = nn.Conv2d(in_channels, out_channels, kernel_size, padding=padding, bias=conv_bias)
+        norm = nn.BatchNorm2d(out_channels)
+        super(ConvBlock, self).__init__(conv, norm)
+
+
+class BaseDecodeHead(nn.Module, metaclass=ABCMeta):
```

Comment:
are decode_head.py, decode_head/decode_head.py, and decoder/decode_head.py identical (files nearby as well)?